client/my-sites/plans-features-main/index.tsx

```diff
@@ -849,6 +854,7 @@ const PlansFeaturesMain = ( {
 										enableFeatureTooltips={ ! isTrailMapCopy }
 										enableCategorisedFeatures={ isTrailMapStructure }
 										featureGroupMap={ isTrailMapStructure ? featureGroupMap : undefined }
+										experimentCallbacks={ { isTrailMapVariant: () => isTrailMapCopy } }
```

Comment:
```suggestion
										experimentCallbacks={ { isTrailMapVariant: () => isAnyTrailMapTreatment } }
```

Not sure if this PR is ready for review or not, but flagging this just in case.

Reply:
Good catch I just put this up for a discussion about the approach. Hadn't tested it thoroughly.